api/src/main/java/org/openmrs/module/eptsreports/metadata/HivMetadata.java

```diff
@@ -419,6 +419,14 @@ public Concept getCD4PercentConcept() {
     return getConcept(uuid);
   }
 
+  // concept_id=307
```

Comment:
Any Tb related metadata that is only used in TB queries should be created in TbMetadata.java, if such metadata are shared across the application, we would create those in CommonMetadata.java